sktime/datasets/base.py

```diff
@@ -614,3 +615,56 @@ def load_airline():
     data.index = pd.Int64Index(data.index)
     data.name = name
     return data
+
+def load_uschange(y_col='Consumption'):
```

Comment:
I'd call it `y_name`, please add it to the doc string 
```suggestion
def load_uschange(y_name="Consumption"):
```